docs/rules/space-before-blocks.md

```diff
@@ -5,6 +5,8 @@ While it is a personal preference where to put the opening brace of blocks,
 it should be consistent across a whole project.
 Having an inconsistent style distracts the reader from seeing the important parts of the code.
 
+**Fixable:** This rule is automatically fixable using the `--fix` flag on the command line.
```

Comment:
I have not given this any thought as I just copied from rule to rule just to be consistent.